recipes/r-freesurferformats/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = '0.1.17' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-freesurferformats
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/freesurferformats_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/freesurferformats/freesurferformats_{{ version }}.tar.gz
+  sha256: 9619a2e882521703a6c58f5e809195f053dd5cd87e0966d0a59f25fe52afbb88
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-pkgfilecache >=0.1.1
+    - r-rmarkdown
+    - r-xml2
+  run:
+    - r-base
+    - r-pkgfilecache >=0.1.1
+    - r-rmarkdown
+    - r-xml2
+
+test:
+  commands:
+    - $R -e "library('freesurferformats')"           # [not win]
+    - "\"%R%\" -e \"library('freesurferformats')\""  # [win]
+
+about:
+  home: https://github.com/dfsp-spirit/freesurferformats
+  license: MIT
+  summary: |
+    Provides functions to read and write neuroimaging data in various file formats, with
+    a focus on 'FreeSurfer' <http://freesurfer.net/> formats. This includes, but is
+    not limited to, the following file formats: 1) MGH/MGZ format files, which can contain
+    multi-dimensional images or other data. Typically they contain time-series of three-dimensional
+    brain scans acquired by magnetic resonance imaging (MRI). They can also contain
+    vertex-wise measures of surface morphometry data. The MGH format is named after
+    the Massachusetts General Hospital, and the MGZ format is a compressed version of
+    the same format. 2) 'FreeSurfer' morphometry data files in binary 'curv' format.
+    These contain vertex-wise surface measures, i.e., one scalar value for each vertex
+    of a brain surface mesh. These are typically values like the cortical thickness
+    or brain surface area at each vertex. 3) Annotation file format. This contains a
+    brain surface parcellation derived from a cortical atlas. 4) Surface file format.
+    Contains a brain surface mesh, given by a list of vertices and a list of faces.
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - dipterix
+    - dfsp-spirit
```

Comment:
Additional maintainers must comment that they wish to be included.

Reply:
I wish to be included.